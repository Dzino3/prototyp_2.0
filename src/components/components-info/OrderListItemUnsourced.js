import React from 'react'

function OrderListItemUnsourced({item, t}) {
  return (
    <div className='orderListItemUnsourced'>
      
      <div className='block'>
        <p className='p2'>{t('listItemULabel2')} </p>
        <p>{item.ORDER_ENTRY_DATE}</p>
        <p className='p2'>{t('listItemULabel3')} </p>
        <p>{item.ORDER_SOURCE_DATE}</p>
      </div>
      <div className='block'>
        <p className='p2'>{t('listItemULabel4')} </p>
        <p>{item.ORDER_STATUS}</p>
        <p className='p2'>{t('listItemULabel5')} </p>
        <p>{item.REQ_SHIP_DATE}</p>
      </div>
      <div className='block'>
        <p className='p2'>{t('listItemULabel6')} </p>
        <p>{item.PRES_DATE}</p>
        <p className='p2'>{t('listItemULabel7')} </p>
        <p>{item.BASE_CODE_DESCR}</p>
      </div>
      <div className='block'>
        <p className='p2'>{t('listItemULabel8')} </p>
        <p>{item.QTY}</p>
      </div>
    </div>
  )
}

export default OrderListItemUnsourced
