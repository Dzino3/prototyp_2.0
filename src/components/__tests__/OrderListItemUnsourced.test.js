import React from "react";
import renderer from 'react-test-renderer';
import OrderListItemUnsourced from '../components-info/OrderListItemUnsourced';

describe(`Rendering OrderListItemSourced`, () => {

    test('should render OrderListItemSourced with item UNSOURCED', function () {
        let itemX = {
            item: "UNSOURCED"
        }
        let tree = renderer.create(<OrderListItemUnsourced item={itemX}></OrderListItemUnsourced>);

        expect(tree.toJSON()).toMatchSnapshot();
    });
})