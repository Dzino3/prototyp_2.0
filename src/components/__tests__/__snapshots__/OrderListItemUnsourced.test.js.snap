// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering OrderListItemSourced should render OrderListItemSourced with item UNSOURCED 1`] = `
<div
  className="orderListItemUnsourced"
>
  <div
    className="block"
  >
    <p
      className="p2"
    >
      Factory delivery date: 
    </p>
    <p />
    <p
      className="p2"
    >
      Order entry date: 
    </p>
    <p />
    <p
      className="p2"
    >
      Order source date: 
    </p>
    <p />
  </div>
  <div
    className="block"
  >
    <p
      className="p2"
    >
      Order status: 
    </p>
    <p />
    <p
      className="p2"
    >
      Req. ship date: 
    </p>
    <p />
  </div>
  <div
    className="block"
  >
    <p
      className="p2"
    >
      Pres. date: 
    </p>
    <p />
    <p
      className="p2"
    >
      Base Code Desciption: 
    </p>
    <p />
  </div>
  <div
    className="block"
  >
    <p
      className="p2"
    >
      Qty.: 
    </p>
    <p />
  </div>
</div>
`;
